tornjak-frontend/src/components/cluster-edit.js

```diff
@@ -210,6 +212,30 @@ class ClusterEdit extends Component {
     }
   }
 
+  deleteCluster() {
+    var cluster = this.state.clusterName;
+    var inputData =
+    {
+      "cluster": {
+        "name": cluster
+      }
+    }
+    if (cluster === "") {
+      return window.alert("Please Choose a Cluster!");
+    }
+    var confirm = window.confirm("Are you sure you want to delete the cluster?");
+    if (!confirm) {
+      return
+    }
+    if (IsManager) {
+      this.TornjakApi.clusterDelete(this.props.globalServerSelected, inputData, this.props.clustersListUpdateFunc, this.props.globalClustersList);
+    } else {
+      this.TornjakApi.localClusterDelete(inputData, this.props.clustersListUpdateFunc, this.props.globalClustersList);
+    }
+    window.alert("CLUSTER DELETED SUCCESSFULLY!");
```

Comment:
how do you know deletion is successful? How do you know if its successfully run?